docs/install.rst

```diff
@@ -46,14 +54,63 @@ For advanced image processing you will need one or several of these packages. Fo
 - `Scikit Image`_ may be needed for some advanced image manipulation.
 - `OpenCV 2.4.6`_ or more recent (provides the package ``cv2``) or more recent may be needed for some advanced image manipulation.
 
-If you are on linux, these packages will likely be in your repos.
+If you are on Linux, these packages will likely be in your repos.
+
+
+Custom paths to external tools
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There are a couple of environment variables used by MoviePy that allow you
+to configure custom paths to the external tools. 
+
+To setup any of these variables, the easiest way is to do it in Python before
+importing objects from MoviePy. For example:
+
+.. code-block:: python
+
+    import os
+    os.environ["FFMPEG_BINARY"] = "my/custom/ffmpeg"
+
+    from moviepy.editor import VideoFileClip
+
+Alternatively, after installing the optional dependencies, you can create
+a ``.env`` file in your working directory that will be automatically read.
+
+There are 2 available environment variables:
+
+``FFMPEG_BINARY``
+    Normally you can leave this one to its default ('ffmpeg-imageio') in which
+    case imageio will download the right ffmpeg binary (on first use) and then always use that binary.
+
+    The second option is ``auto-detect``. In this case ffmpeg will be whatever
+    binary is found on the computer: generally ``ffmpeg`` (on Linux/macOS) or ``ffmpeg.exe`` (on Windows).
+
+    Lastly, if you want to use a binary at a special location on your disk, enter it like this::
+
+        FFMPEG_BINARY = r"path/to/ffmpeg" # on linux
+        FFMPEG_BINARY = r"path\to\ffmpeg.exe" # on windows
```

Comment:
It seems a little strange leaving the syntax like this when the user is no longer writing them as Python variables. Presumably it should be like

    os.environ["FFMPEG_BINARY"] = "path/to/ffmpeg"

or like the actual contents of the `.env` file:

    FFMPEG_BINARY="path/to/ffmpeg"

Either of them might need the 2 versions (linux and windows), I'm not sure.

The same applies to ImageMagick on line 102.